src/Datadog.Trace/Span.cs

```diff
@@ -290,10 +288,15 @@ public void SetException(Exception exception)
                     exception = aggregateException.InnerExceptions[0];
                 }
 
+                Status = SpanStatus.Error.WithDescription(exception.Message);
                 SetTag(Trace.Tags.ErrorMsg, exception.Message);
                 SetTag(Trace.Tags.ErrorStack, exception.ToString());
                 SetTag(Trace.Tags.ErrorType, exception.GetType().ToString());
             }
+            else
+            {
+                Status = SpanStatus.Error;
```

Comment:
I quickly browsed all 74 references. Looks like it is never null.
Even if we decide to change it, I prefer to do it in a separate PR. So that we can easily and quickly revert it if it will not work.